source/runbooks/oracle-licence-discovery.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+owner_slack: "#modernisation-platform"
+title: oracle licence discovery
+last_reviewed_on: 2024-12-05
+review_in: 6 months
+---
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-NXTCMQ7ZX6"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-NXTCMQ7ZX6');
+</script>
+
+# <%= current_page.data.title %>
+
+# Centrally Track Oracle Database Licences in AWS Organisations
+
+## Introduction
+
+AWS Licence Manager and AWS Systems Manager can be used together to centrally track Oracle database Licences across an AWS Organization. This approach helps ensure compliance and optimise Licence usage.
+
+### Where to find the oracle Licence discovery
+
+The oracle Licence discovery can be found in the organisation security account it is broken down into the following elements:
+
+### AWS Licence Manager
+AWS Licence Manager makes it easier to manage your software Licences from vendors such as Microsoft, SAP, Oracle, and IBM across AWS and on-premises environments. It helps you manage, track, and control software Licences to ensure compliance and reduce costs.
+
+To find out the state of the Licence manager go to the "AWS Licence Manager" and under the section  "self managed Licences" in the AWS console.
+   - look for a self managed licence called "OracleDbEELicenceConfiguration"
+   - this will show you the EC2 Licence usage for oracle databases.
+
+![List of EC2 instances](images/Licence_manager_setup.png) 
+
+### AWS Systems Manager
+AWA Systems Manager helps you automate operational tasks across your AWS resources. You can create automation workflows, manage patch compliance, and configure operating systems.
+
+To look at the automation document or the automation executions go to the "Systems Manager" section in the AWS console.
+   - The automation documents are stored in the "documents" section under the self owned tab.
+
+   ![automation documents location ](images/documents_section.png)
+
+   - To view the automation, click on the automation section on the left hand side and click on the execution id next to the runbook name of "OracleDbLTS-Orchestrate" from here you will be able to see the successful and failed executions and dig down further.
+
+   ![automation location ](images/automation_section.png)
+
+   - To find the state associated with automation, go to the "State Manager" section under Systems Manager this will show you the last run state of the automation and when the frequency of the automation, you can also apply the association from here to force a run of the automation.
```

Comment:
automation. You can also apply the association from here to force a run of the automation.